data.sql

```diff
@@ -1,5 +1,4 @@
-/* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Agumon', '2023-2-15', 0, true, 10.23);
```

Comment:
- [ ] Kindly insert the correct information from the requirement, please correct the date of birth, it should be 2013-2-3 instead of 2023-2-15 👇🏽 

> Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.